### PR TITLE
newsfeed intro image [redo]

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #11402 .
redo for @rdeutz as it was merged in the wrong brach

### CODE REVIEW

File: components/com_content/views/category/view.feed.php [hunk 1]
```diff
@@ -34,10 +34,19 @@ class ContentViewCategory extends JViewCategoryfeed
...
+		$item->description = '';
+		$obj = json_decode($item->images);
+		$introImage = ( isset( $obj->{'image_intro'} ) ) ? $obj->{'image_intro'} : '';
+
+		if (isset($introImage) && ($introImage != "")) 
+		{
+			$image = preg_match('/http/', $introImage)? $introImage : JURI::root() . $introImage;
+			$item->description = '<p><img src="' . $image . '" /></p>';
+		}
+		$item->description .= ($params->get('feed_summary', 0) ? $item->introtext . $item->fulltext : $item->introtext);
```
Comment (yvesh) [inline components/com_content/views/category/view.feed.php]: You still need a new line after the closing if braces :-) (in view.feed.php too)

**Comment by brianteeman** [comment]
This PR was RTC and was merged by @rdeutz into the wrong branch so I had to redo it

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11687">issues.joomla.org/joomla-cms/11687</a>.</sub>

**Comment by wilsonge** [comment]
Merged with https://github.com/joomla/joomla-cms/commit/4058aa646152abfc5289fa02f66911c6192d9b22

**Comment by brianteeman** [comment]
Thanks

**Pull request closed without merging**